src/compas_fab/robots/robot_library.py

```diff
@@ -0,0 +1,180 @@
+        semantics = RobotSemantics.from_srdf_file(srdf_filename, model)
+
+        if load_geometry:
+            loader = LocalPackageMeshLoader(compas_fab.get(local_package_mesh_folder), "")
+            model.load_geometry(loader)
+
+        robot = RobotClass(model, semantics=semantics)
+
+        if client:
+            robot.client = client
+
+        return robot
+
+    @classmethod
+    def rfl(cls, client=None, load_geometry=True):
+        """Create and return the RFL robot with 4 ABB irb 4600 and twin-gantry setup.
+
+        The returned :class:`compas_fab.robots.Robot` object contains the robot model and semantics.
+
+        Parameters
+        ----------
+        client: object
+            Backend client. Default is `None`.
+        load_geometry: bool, optional
+            Default is `True`, which means that the geometry is loaded.
+            `False` can be used to speed up the creation of the robot.
+
+        Returns
+        -------
+        :class:`compas_fab.robots.Robot`
+            Newly created instance of the robot.
+        """
+
+        robot = cls._load_library_model(
+            urdf_filename=compas_fab.get("robot_library/rfl/urdf/robot_description.urdf"),
+            srdf_filename=compas_fab.get("robot_library/rfl/robot_description_semantic.srdf"),
+            local_package_mesh_folder="robot_library/rfl",
+            client=client,
+            load_geometry=load_geometry,
+        )
+
+        return robot
+
+    @classmethod
+    def ur5(cls, client=None, load_geometry=True):
+        """Returns a UR5 robot.
+
+        The returned :class:`compas_fab.robots.Robot` object contains the robot model and semantics.
+
+        Parameters
+        ----------
+        client: object
+            Backend client. Default is `None`.
+        load_geometry: bool, optional
+            Default is `True`, which means that the geometry is loaded.
+            `False` can be used to speed up the creation of the robot.
+
+        Returns
+        -------
+        :class:`compas_fab.robots.Robot`
+            Newly created instance of the robot.
+        """
+
+        robot = cls._load_library_model(
+            urdf_filename=compas_fab.get("robot_library/ur5_robot/urdf/robot_description.urdf"),
+            srdf_filename=compas_fab.get("robot_library/ur5_robot/robot_description_semantic.srdf"),
+            local_package_mesh_folder="robot_library/ur5_robot",
+            client=client,
+            load_geometry=load_geometry,
+        )
+
+        return robot
+
+    @classmethod
+    def ur10e(cls, client=None, load_geometry=True):
+        """Returns a UR10e robot.
+
+        The returned :class:`compas_fab.robots.Robot` object contains the robot model and semantics.
+
+        Parameters
+        ----------
+        client: object
+            Backend client. Default is `None`.
+        load_geometry: bool, optional
+            Default is `True`, which means that the geometry is loaded.
```

Comment:
I like the fact that `load_geometry` defaults to True. Let's change it also in the other classes that already have functions like that

Reply:
Can you remind me which are those functions? Maybe lets put it to a new issue / PR.

Reply:
We decided to do this in a separate PR instead of mixing it with this.